### PR TITLE
Add an example of FastViT model (Infernece)

### PR DESCRIPTION
**Pull request by cfgfung**
# What does this PR do?
Added the example code of the validated FastViT.

Original performance:
n_iterations: 20
Total latency (ms): 451.754093170166
Average latency (ms): 22.5877046585083

BF16 + Graph mode:
n_iterations: 20
Total latency (ms): 47.97697067260742
Average latency (ms): 2.398848533630371

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by yeonsily** [comment]
@cfgfung Is this the same change with https://github.com/huggingface/optimum-habana/pull/814?

File: examples/object-segementation/ClipSeg/run_example.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    image = Image.open(requests.get(args.image_path, stream=True).raw)
+    texts = []
+    for text in args.prompt.split(','):
+        texts.append(text)
+
+    if args.use_hpu_graphs:
+        model = ht.hpu.wrap_in_hpu_graph(model)
+
+    autocast = torch.autocast(device_type="hpu", dtype=torch.bfloat16, enabled=args.bf16)
+    model.to("hpu")
```
Comment (yeonsily) [inline examples/object-segementation/ClipSeg/run_example.py:85]: don't we need model.eval()?

Comment (cfgfung) [reply examples/object-segementation/ClipSeg/run_example.py:85]: torch.no_grad() is used in line 87. The model will behave the same as model.eval().

**Comment by cfgfung** [comment]
> @cfgfung Is this the same change with #814?

@yeonsily This is another example. Look similar but not the same.

**Comment by cfgfung** [comment]
Add GPU Benchmark result:

A100 - 80GB , CUDA 12.2 , NVIDIA-SMI 535.54.03
With BF16 Autocast:
n_iterations: 20
Total latency (ms): 138.54217529296875
Average latency (ms): 6.9271087646484375

**Comment by cfgfung** [comment]
Hi,

Some updates here:
- Tested with 'make style' to align the code styles.
- Added and updated related readmes.
- Intentionally not adding test cases as this is a simple example
- This is a TIMM model and does not belong to Transformer module. The current existing codes in run_image_classification.py are not compatible with TIMM model. Therefore I separate this into another file. run_timm_example.py.

**Comment by libinta** [comment]
@cfgfung  can you rebase and provide the latest test result. thanks

**Comment by cfgfung** [comment]
> @cfgfung can you rebase and provide the latest test result. thanks

Hi, 
I have applied rebase on this branch and added test cases.

ViT_TestCases

**Comment by yeonsily** [comment]
LGTM. Just can you please make sure if the CI pass on the synapse 1.16 release?

**Comment by cfgfung** [comment]
> LGTM. Just can you please make sure if the CI pass on the synapse 1.16 release?

Hi,
I have ran the test on Synapse 1.16 and it passed.

image

**Review by regisss — APPROVED** [review]

**Pull request merged**